ansible/roles/debops.postldap/defaults/main.yml

```diff
@@ -0,0 +1,627 @@
+# .. envvar:: postldap__ldap_default_virtual_domain_rdn [[[
+#
+# The rdn of the default virtual mail domain used by Postfix.
+# This is usually the same domain used by the LDAP.
+postldap__ldap_default_virtual_domain_rdn: '{{ ansible_domain }}'
+                                                                   # ]]]
+# .. envvar:: postldap__ldap_default_virtual_domain_dn [[[
+#
+# The dn of the default virtual mail domain used by Postfix.
+postldap__ldap_default_virtual_domain_dn: '{{ (["dc="
+                                                           + postldap__ldap_default_virtual_domain_rdn]
+                                                           + postldap__ldap_domains_dn) | join(",") }}'
+                                                                   # ]]]
+                                                                   # ]]]
+# LDAP connection options [[[
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+# .. envvar:: postldap__ldap_server_uri [[[
+#
+# The URI address of the LDAP server used by Postfix.
+postldap__ldap_server_uri: '{{ (ansible_local.ldap.uri
+                                            if (ansible_local|d() and ansible_local.ldap|d() and
+                                                ansible_local.ldap.uri|d())
+                                            else [""]) | first }}'
+
+                                                                   # ]]]
+# .. envvar:: postldap__ldap_server_port [[[
+#
+# The TCP port which should be used for connections to the LDAP server.
+postldap__ldap_server_port: '{{ ansible_local.ldap.port
+                                            if (ansible_local|d() and ansible_local.ldap|d() and
+                                                ansible_local.ldap.port|d())
+                                            else ("389" if postldap__ldap_start_tls|bool else "636") }}'
+
+                                                                   # ]]]
+# .. envvar:: postldap__ldap_start_tls [[[
+#
+# If ``True``, Postfix will use STARTTLS extension to make encrypted
+# connections to the LDAP server.
+postldap__ldap_start_tls: '{{ ansible_local.ldap.start_tls
+                                          if (ansible_local|d() and ansible_local.ldap|d() and
+                                              (ansible_local.ldap.start_tls|d())|bool)
+                                          else True }}'
+
+                                                                   # ]]]
+                                                                   # ]]]
+# LDAP settings [[[
+# ^^^^^^^^^^^^^^^^^
+
+# .. envvar:: postldap__ldap_user_filter [[[
+#
+# The LDAP filter used to look up user accounts in the directory.
+postldap__ldap_user_filter: '(&
+                                           (objectClass=inetOrgPerson)
+                                           (objectClass=postfixUser)
+                                         )
+                                         (mailEnabled=TRUE)
```

Comment:
I think that without the custom mail schema which is currently planned but not available yet, and with `postfixUser` object class and `mailEnabled` attribute defined as required, no user accounts will be found by the LDAP filter, even with the required `authorizedService` values. Perhaps you could drop the custom object class and attribute from the search filter for now, at least until the proper mail schema is developed?